### PR TITLE
added missing alt text

### PR DESCRIPTION
**Pull request by v-emilypr**

### CODE REVIEW

File: iis/configuration/system.applicationHost/sites/site/ftpServer/index.md [hunk 1]
```diff
@@ -253,14 +253,14 @@ To support FTP publishing for your Web server, you must install the FTP service.
...
-        [![](index/_static/image22.png)](index/_static/image21.png)
+        ![Screenshot of the Add F T P Site window displaying the Binding and S S L Settings page. ](index/_static/image21.png)
 6. On the next page of the wizard:
 
     - Select **Basic** for the **Authentication** settings.
     - For the **Authorization** settings, choose &quot;Specified users&quot; from the **Allow access to** drop-down, and enter an account name in the box below the drop-down menu.
     - Select **Read** and **Write** for the **Permissions** option.
     - When you have completed these items, click **Finish**.  
-        [![](index/_static/image24.png)](index/_static/image23.png)
+        ![Screenshot of the Add F T P Site windows displaying the Authentication and Authorization Information page.](index/_static/image23.png)
```
Comment (MeeraDi) [inline iis/configuration/system.applicationHost/sites/site/ftpServer/index.md:263]: Screenshot of the Add F T P Site **window** displaying the Authentication and Authorization Information page.

File: iis/configuration/system.webServer/tracing/traceProviderDefinitions/add/areas/add.md [hunk 2]
```diff
@@ -47,7 +47,7 @@ After you finish the default installation of IIS 7 and later, you must install t
 1. On the **Start** screen, move the pointer all the way to the lower left corner, right-click the **Start** button, and then click **Control Panel**.
 2. In **Control Panel**, click **Programs and Features**, and then click **Turn Windows features on or off**.
 3. Expand **Internet Information Services**, expand **World Wide Web Services**, expand **Health and Diagnostics**, and then select **Tracing**.  
-    [![](add/_static/image4.png)](add/_static/image3.png)- Click **OK**.
+    ![Screenshot of the Windows Features dialgog box. Tracing is highlighted in the expanded menu.](add/_static/image3.png)- Click **OK**.
```
Comment (MeeraDi) [inline iis/configuration/system.webServer/tracing/traceProviderDefinitions/add/areas/add.md:50]: Screenshot of the Windows Features **dialog** box.

File: iis/install/web-platform-installer/using-the-microsoft-web-platform-installer.md [hunk 3]
```diff
@@ -82,21 +82,21 @@ Web PI simplifies the discovery and installation of Web server components by:
 
 To see all the IIS components available for installation, just search for **IIS** or to install the recommended set, search for and install the **IIS 7 Recommended Configuration**
 
-![](using-the-microsoft-web-platform-installer/_static/image10.png)
+![Screenshot of the Web Platform Installer 3 displaying the Search results for i i s page.](using-the-microsoft-web-platform-installer/_static/image10.png)
```
Comment (MeeraDi) [inline iis/install/web-platform-installer/using-the-microsoft-web-platform-installer.md:85]: Screenshot of the Web Platform Installer 3 displaying the Search results for **I I S** page.

File: iis/configuration/system.webServer/tracing/traceProviderDefinitions/add/index.md [hunk 4]
```diff
@@ -66,7 +66,7 @@ After you finish the default installation of IIS 7 and later, you must install t
 2. In **Control Panel**, click **Programs and Features**, and then click **Turn Windows Features on or off**.
 3. Expand **Internet Information Services**, then **World Wide Web Services**, then **Health and Diagnostics**.
 4. Select **Tracing**, and then click **OK**.  
-    [![](index/_static/image8.png)](index/_static/image7.png)
+    ![Screenshot of the Windows Features dialog box with the Tracing feature is highlighted.](index/_static/image7.png)
```
Comment (MeeraDi) [inline iis/configuration/system.webServer/tracing/traceProviderDefinitions/add/index.md:69]: Screenshot of the Windows Features dialog box with the Tracing feature highlighted.

**Comment by v-emilypr** [comment]
@Rick-Anderson, @mcleblanc
This PR addresses missing alt text and is ready for final review.

**Review by Rick-Anderson — APPROVED** [review]

**Pull request merged**